packages/number-field/README.md

```diff
@@ -267,3 +267,15 @@ The `<sp-number-field>` component doesn't manage a default value by itself. This
         });
     });
 </script>
+
+## States
+
+Use the `required` attribute to indicate a number field value is required. Dictate the validity or invalidity state of the text entry with the `valid` or `invalid` attributes.
```

Comment:
copy pasta'd from text field. this readme is still on the docs audit list so didn't do a full pass.